paddle/fluid/framework/new_executor/interpretercore.cc

```diff
@@ -235,19 +292,33 @@ paddle::framework::FetchList InterpreterCore::Run(
 
   if (!is_build_) {
     LOG_FIRST_N(INFO, 1) << "New Executor is Running.";
-    paddle::framework::interpreter::BuildVariableScope(
-        block_, execution_config_, &var_scope_);
+    if (FLAGS_enable_new_ir_in_executor) {
+      ::ir::BuildScope(
+          ir_program_->block(), local_scope_, &value_2_var_name_map_);
+    } else {
+      paddle::framework::interpreter::BuildVariableScope(
```

Comment:
```suggestion
      interpreter::BuildVariableScope(
```

这里其实不需要加`paddle::framework` 的，太长了

Reply:
done